src/app/BreadcrumbPage/BreadcrumbPage.tsx

```diff
@@ -52,8 +61,8 @@ export interface BreadcrumbTrail {
 
 export const BreadcrumbPage: React.FC<BreadcrumbPageProps> = (props) => {
   return (
-    <>
-      <PageSection>
+    <PageGroup>
+      <PageBreadcrumb>
```

Comment:
Sorry, can we revert these changes? Seems like this wrapper adds a white background which is quite weird :))

![Screenshot from 2023-03-28 16-02-32](https://user-images.githubusercontent.com/68053619/228353998-203653d7-97e3-4017-b444-0e7433143f36.png)